mysql-test/suite/rpl/t/percona_bug1441259-slave.opt

```diff
@@ -0,0 +1,3 @@
+--read_only=1
```

Comment:
Better set these dynamically in the testcase itself, to avoid server restart (and to make the testcase immediately applicable to the current 5.7)